department-of-veterans-affairs/abd-vro
# MCP-2528 Remove Sufficiency workaround

## What was the problem?

Sufficiency workaround is no longer needed since it is not treated as an error

Associated tickets or Slack threads:

- [MCP-2528](https://amida.atlassian.net/browse/MCP-2528)

## How does this fix it?[^1]

Remove the code

## How to test this PR
- Check that E2E still passes


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
